website/app.py

```diff
@@ -110,7 +111,13 @@ def init_app(settings_module='website.settings', set_backends=True, routes=True,
     settings = importlib.import_module(settings_module)
 
     build_log_templates(settings)
-    init_addons(settings, routes)
+    try:
+        init_addons(settings, routes)
+    except AssertionError:
+        if supress_assertion_errors:
```

Comment:
Not sure if this is right place to do this. @chrisseto moved the exception handling to the `process_rules` function, so the option to catch the error should probably go in the same place.